### ISSUE TITLE
Introduce a third panel for tours

### ISSUE THREAD

**Issue by alexsegura** [coopcycle/coopcycle-web#3595]
See #3552

This would solve most of the problems of tour edition (reordering, drag'n'dropping a new task), also in terms of implementation with `react-beautiful-dnd` draggable/droppable structure.

**Comment by ERAMANrepartos** [comment]
@alexsegura This is completely true. However, to implement such a broad feature, there needs to be significantly more information here. Please elaborate about what this third dashboard panel would involve and how it many be represented.

**Comment by jimakker** [comment]
This already has a PR but I'm using that branch for https://github.com/coopcycle/coopcycle-web/issues/3597. I will discard the PR and we'll merge the other one.
